### DISCUSSION TITLE
Can a Chapter Tag be centered?

### DISCUSSION THREAD

**Discussion by orez-ero** [vkbo/novelWriter#1660] [Q&A]
When I export a novelwriter project to an .odt file, the chapter heading is left-aligned. I'd like it to be centered. I have tried to use the centering tags (>><<) both before and after the chapter tag (H2), but neither seems to work. ## >>C hapter One << gives a left-aligned ">> Chapter One <<" while >> ## Chapter One <<  gives a correctly centered (but un-bolded) "## Chapter One." Is there a _correct_ way to center a chapter heading?

**Comment by vkbo** [comment]
Hi, yeah, the `>> <<` codes only work on text paragraphs. But I can add an option to centre text for headers in the build settings.

I have created a new feature request for this at #1661

**Reply by vkbo** [reply]
Please use the reply feature to respond to a post so the reply stays in the correct thread. #1393.

Anyway:

> That would be great! Thank you! Also, are the build settings accessible to us, the users? I haven't found them, but I'll have another look. 

See https://novelwriter.io/docs/project_manuscript.html#build-settings
